### ISSUE TITLE
[modal] Focus management

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/OR-UK-frontend#190]
URL: https://openreferraluk.org/dashboard

When the dialog appears on the screen, keyboard focus (whose control depends upon the dialogs purpose) should be moved to the default focusable control inside the dialog. This would be the close button in this case.

After the dialog is dismissed, keyboard focus should be moved back to where it was before it moved into the dialog (Read button).

**Comment by kingdomlevel** [comment]
Fix to the dashboard screen - depends on #209 being merged

**Comment by kingdomlevel** [comment]
PR #209 has now been merged - removing the 'blocked' label and moving to 'To Do'
